docs/CHANGELOG.asciidoc

```diff
@@ -39,6 +40,395 @@ This section summarizes the changes in each release.
+* [Hosts] Endpoint overview on host details page {kibana-pull}71466[#71466]
+* [Detection Engine] - Update exceptions logic {kibana-pull}71512[#71512]
+* [Detection Engine][Lists] Adds the ability for exception lists to be multi-list queried {kibana-pull}71540[#71540]
+* [DETECTIONS] Set rule status to failure only on large gaps {kibana-pull}71549[#71549]
+* [Lists] - Update exception comments logic in API {kibana-pull}71602[#71602]
+* [Exceptions] - Adds filtering to endpoint index patterns by exceptional fields {kibana-pull}71757[#71757]
+* Full screen timeline, Collapse event {kibana-pull}71786[#71786]
+* [Detection Engine][Lists] Adds specific endpoint_list REST API and API for abilities to auto-create the endpoint_list if it gets deleted {kibana-pull}71792[#71792]
+* [Detections] Associate Endpoint Exceptions List to Rule during rule creation/update {kibana-pull}71794[#71794]
+* [Detections] Disable exceptions for Threshold and ML rules {kibana-pull}72137[#72137]
+* [Detection Engine][Lists] Adds list permissions {kibana-pull}72335[#72335]
+* [Exceptions] - Make esTypes and subType available to index patterns {kibana-pull}72336[#72336]
+* [Detection Engine][Lists] Adds conflict versioning and io-ts improvements to lists {kibana-pull}72337[#72337]
+* [Exceptions] - Update UI exceptions builder nested logic {kibana-pull}72490[#72490]
+* [Detection Engine][Lists] Adds version and immutability data structures {kibana-pull}72730[#72730]
+* [Exceptions] - Require non empty entries and non empty string values in exception list items {kibana-pull}72748[#72748]
+* [Exceptions] - Update exception item comments to include id {kibana-pull}73129[#73129]
+* [Exceptions] - Update rule.exceptions_list to include exception list list_id {kibana-pull}73349[#73349]
+* [Exceptions] - Updates exception hooks and viewer {kibana-pull}73588[#73588]
+* [Exceptions] - Update how nested entries are displayed in exceptions viewer {kibana-pull}73745[#73745]
+* [Exceptions] Adds autocomplete workaround for .text fields {kibana-pull}73761[#73761]
+* [Endpoint][Exceptions] Only write manifest to policy when there are changes {kibana-pull}72000[#72000]
+* [Exceptions] Use semantic version for manifest version + Scaling Tweaks {kibana-pull}73388[#73388]
+* Resolver in timeline {kibana-pull}69728[#69728]
+Uptime::
+* Uses ML Capabilities API to determine license type {kibana-pull}66921[#66921]
+* Landing page for Observability {kibana-pull}67467[#67467]
+* Monitor availability reporting {kibana-pull}67790[#67790]
+* Creates "Add data" links in all Observability app headers {kibana-pull}69016[#69016]
+* Observability overview page {kibana-pull}69141[#69141]
+* Availability alert {kibana-pull}70284[#70284]
+* Duration Anomaly Alert {kibana-pull}71208[#71208]
+
+[float]
+[[bug-v7.9.0]]
+=== Bug fixes
+Alerting::
+* Adds lazy loading of alerting UI components {kibana-pull}65060[#65060]
+* Migrates the old `alerting` consumer to be `alerts` {kibana-pull}69982[#69982]
```

Comment:
This one can be removed as the PR got reverted (https://github.com/elastic/kibana/pull/73420)